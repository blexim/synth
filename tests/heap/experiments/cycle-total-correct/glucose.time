WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 20
3.24user 0.05system 0:03.35elapsed 98%CPU (0avgtext+0avgdata 114140maxresident)k
0inputs+16outputs (0major+26579minor)pagefaults 0swaps
