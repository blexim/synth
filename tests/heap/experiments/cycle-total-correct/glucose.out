c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:        153783                                                                   |
c |  Number of clauses:          613657                                                                   |
c |  Parse time:                   0.19 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           5.89 Mb                                                                |
c |  Simplification time:          0.90 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c |       44         0      227 |   33633   237114  1286665 |     2     5930      507     3684 |  6.471 % |
c =========================================================================================================
c restarts              : 72 (220 conflicts in avg)
c blocked restarts      : 43 (multiple: 8) 
c last block at restart : 72
c nb ReduceDB           : 3
c nb removed Clauses    : 7282
c nb learnts DL2        : 595
c nb learnts size 2     : 286
c nb learnts size 1     : 69
c conflicts             : 15907          (4903 /sec)
c decisions             : 39902          (0.00 % random) (12299 /sec)
c propagations          : 15663502       (4828065 /sec)
c conflict literals     : 431968         (36.90 % deleted)
c nb reduced Clauses    : 133
c CPU time              : 3.24426 s

s UNSATISFIABLE
